geomesa-fs/geomesa-fs-storage/geomesa-fs-storage-convert/src/main/scala/org/locationtech/geomesa/fs/storage/converter/ConverterStorageFactory.scala

```diff
@@ -29,9 +34,22 @@ class ConverterStorageFactory extends FileSystemStorageFactory with LazyLogging
       }
       SimpleFeatureConverter(metadata.sft, converterConfig)
     }
-    new ConverterStorage(context, metadata, converter)
-  }
 
+    val pathFilteringOpts =
+      context.conf.getValByRegex(Pattern.quote(PathFilterOptsPrefix) + ".*").asScala.map {
+        case (k, v) => k.substring(PathFilterOptsPrefix.length) -> v
+      }
+
+    val pathFiltering = Option(context.conf.get(PathFilterName)).flatMap { name =>
+      val factory = PathFilteringFactory.load(NamedOptions(name, pathFilteringOpts.toMap))
+      if (factory.isEmpty) {
+        logger.warn(s"Failed to load ${classOf[PathFiltering].getName} for config '$name'")
```

Comment:
do you think a warning is ok here, or should we throw an error? i worry people might not notice the warning, and then get worse performance but since things still work correctly they might not realize the mistake

Reply:
lgtm otherwise!